### PR TITLE
[stabilization] fix reporting of attitude setpoint in hybrid mode

### PR DESCRIPTION
**Pull request by gautierhattenberger**
also add stab message to flight recorder

### CODE REVIEW

File: sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c [hunk 1]
```diff
@@ -432,11 +433,6 @@ void stabilization_indi_init(void)
  */
 void stabilization_indi_enter(void)
 {
-  /* reset psi setpoint to current psi angle */
-  stab_att_sp_euler.psi = stabilization_attitude_get_heading_i();
```
Comment (dewagter) [inline sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c:436]: Why is this removed?

Comment (gautierhattenberger) [reply sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c:436]: This is meaningless here. `stab_att_sp_euler` is not the global variable used to set command, it is only needed to store the last command value (passed as a parameter of the run function) to send it later over telemetry.
If this reset is still needed (and maybe it is, @EwoudSmeur ?), it as to be done from upper level (guidance or rc_input).

Comment (dewagter) [reply sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c:436]: If when switching back to INDI (which we do e.g. when testing neural controllers) the heading does not jump then it is fine.

**Review by dewagter — APPROVED** [review]
